public/index.html

```diff
@@ -1 +1,17 @@
-
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Calendar-Proxy</title>
+</head>
+<body>
+  <div id="app">NO REACT</div>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+  <script src="http://localhost:3005/bundle.js"></script>
+  <!-- <script>
+    ReactDOM.render(
+      React.createElement(App),document.getElementById('app')
+    );
+  </script> -->
```

Comment:
Is this commented out because there is nothing to render yet?

Reply:
This is because unfortunately, App class is not importing to the browser correctly. So, the ReactDOM.render command from the bundle is being run, it looks for the div named 'app' and works with it.